diff --git a/QuickstartAuthServer.csproj b/BellwoodAuthServer.csproj
similarity index 99%
rename from QuickstartAuthServer.csproj
rename to BellwoodAuthServer.csproj
index 4010145..803fb93 100644
--- a/QuickstartAuthServer.csproj
+++ b/BellwoodAuthServer.csproj
@@ -13,5 +13,4 @@
     <PackageReference Include="Serilog.AspNetCore" Version="8.0.3" />
   </ItemGroup>
 
-
 </Project>
diff --git a/Config.cs b/Config.cs
index b80ca84..9b0f7b9 100644
--- a/Config.cs
+++ b/Config.cs
@@ -3,7 +3,7 @@ using Duende.IdentityServer;
 using Duende.IdentityServer.Models;
 using Duende.IdentityServer.Test;
 
-namespace QuickstartAuthServer;
+namespace BellwoodAuthServer;
 
 public static class Config
 {
diff --git a/HostingExtensions.cs b/HostingExtensions.cs
index 5fc3c71..14c19c1 100644
--- a/HostingExtensions.cs
+++ b/HostingExtensions.cs
@@ -4,7 +4,7 @@ using Serilog;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
 
-namespace QuickstartAuthServer
+namespace BellwoodAuthServer
 {
     internal static class HostingExtensions
     {
diff --git a/Pages/Account/AccessDenied.cshtml b/Pages/Account/AccessDenied.cshtml
index 9b0ab53..a60ec5b 100644
--- a/Pages/Account/AccessDenied.cshtml
+++ b/Pages/Account/AccessDenied.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Account.AccessDeniedModel
+@model BellwoodAuthServer.Pages.Account.AccessDeniedModel
 @{
 }
 <div class="row">
diff --git a/Pages/Account/AccessDenied.cshtml.cs b/Pages/Account/AccessDenied.cshtml.cs
index 6029df3..8e780f0 100644
--- a/Pages/Account/AccessDenied.cshtml.cs
+++ b/Pages/Account/AccessDenied.cshtml.cs
@@ -1,6 +1,6 @@
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Account;
+namespace BellwoodAuthServer.Pages.Account;
 
 public class AccessDeniedModel : PageModel
 {
diff --git a/Pages/Account/Create/Index.cshtml b/Pages/Account/Create/Index.cshtml
index 4872830..0d74a90 100644
--- a/Pages/Account/Create/Index.cshtml
+++ b/Pages/Account/Create/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Create.Index
+@model BellwoodAuthServer.Pages.Create.Index
 
 <div class="login-page">
     <div class="lead">
@@ -28,13 +28,13 @@
                 </div>
                 <div class="form-group">
                     <label asp-for="Input.Email"></label>
-                    <input type="email" class="form-control" placeholder="Email" asp-for="Input.Email" >
+                    <input type="email" class="form-control" placeholder="Email" asp-for="Input.Email">
                 </div>
 
                 <button class="btn btn-primary" name="Input.Button" value="create">Create</button>
                 <button class="btn btn-secondary" name="Input.Button" value="cancel">Cancel</button>
             </form>
         </div>
-        
+
     </div>
 </div>
\ No newline at end of file
diff --git a/Pages/Account/Create/Index.cshtml.cs b/Pages/Account/Create/Index.cshtml.cs
index 39c3ed6..8773196 100644
--- a/Pages/Account/Create/Index.cshtml.cs
+++ b/Pages/Account/Create/Index.cshtml.cs
@@ -6,7 +6,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Create;
+namespace BellwoodAuthServer.Pages.Create;
 
 [SecurityHeaders]
 [AllowAnonymous]
diff --git a/Pages/Account/Create/InputModel.cs b/Pages/Account/Create/InputModel.cs
index 7a42bf0..c718fc7 100644
--- a/Pages/Account/Create/InputModel.cs
+++ b/Pages/Account/Create/InputModel.cs
@@ -1,6 +1,6 @@
 using System.ComponentModel.DataAnnotations;
 
-namespace QuickstartAuthServer.Pages.Create;
+namespace BellwoodAuthServer.Pages.Create;
 
 public class InputModel
 {
diff --git a/Pages/Account/Login/Index.cshtml b/Pages/Account/Login/Index.cshtml
index 13cbaa3..9483b80 100644
--- a/Pages/Account/Login/Index.cshtml
+++ b/Pages/Account/Login/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Login.Index
+@model BellwoodAuthServer.Pages.Login.Index
 
 <div class="login-page">
     <div class="lead">
@@ -13,77 +13,77 @@
 
         @if (Model.View.EnableLocalLogin)
         {
-            <div class="col-sm-6">
-                <div class="card">
-                    <div class="card-header">
-                        <h2>Local Account</h2>
-                    </div>
+        <div class="col-sm-6">
+            <div class="card">
+                <div class="card-header">
+                    <h2>Local Account</h2>
+                </div>
 
-                    <div class="card-body">
-                        <form asp-page="/Account/Login/Index">
-                            <input type="hidden" asp-for="Input.ReturnUrl" />
+                <div class="card-body">
+                    <form asp-page="/Account/Login/Index">
+                        <input type="hidden" asp-for="Input.ReturnUrl" />
 
-                            <div class="form-group">
-                                <label asp-for="Input.Username"></label>
-                                <input class="form-control" placeholder="Username" asp-for="Input.Username" autofocus>
-                            </div>
-                            <div class="form-group">
-                                <label asp-for="Input.Password"></label>
-                                <input type="password" class="form-control" placeholder="Password" asp-for="Input.Password" autocomplete="off">
-                            </div>
-                            
-                            @if (Model.View.AllowRememberLogin)
+                        <div class="form-group">
+                            <label asp-for="Input.Username"></label>
+                            <input class="form-control" placeholder="Username" asp-for="Input.Username" autofocus>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="Input.Password"></label>
+                            <input type="password" class="form-control" placeholder="Password" asp-for="Input.Password" autocomplete="off">
+                        </div>
+
+                        @if (Model.View.AllowRememberLogin)
                             {
-                                <div class="form-group">
-                                    <div class="form-check">
-                                        <input class="form-check-input" asp-for="Input.RememberLogin">
-                                        <label class="form-check-label" asp-for="Input.RememberLogin">
-                                            Remember My Login
-                                        </label>
-                                    </div>
-                                </div>
+                        <div class="form-group">
+                            <div class="form-check">
+                                <input class="form-check-input" asp-for="Input.RememberLogin">
+                                <label class="form-check-label" asp-for="Input.RememberLogin">
+                                    Remember My Login
+                                </label>
+                            </div>
+                        </div>
                             }
-                            
-                            <button class="btn btn-primary" name="Input.Button" value="login">Login</button>
-                            <button class="btn btn-secondary" name="Input.Button" value="cancel">Cancel</button>
-                        </form>
-                    </div>
+
+                        <button class="btn btn-primary" name="Input.Button" value="login">Login</button>
+                        <button class="btn btn-secondary" name="Input.Button" value="cancel">Cancel</button>
+                    </form>
                 </div>
             </div>
+        </div>
         }
 
         @if (Model.View.VisibleExternalProviders.Any())
         {
-            <div class="col-sm-6">
-                <div class="card">
-                    <div class="card-header">
-                        <h2>External Account</h2>
-                    </div>
-                    <div class="card-body">
-                        <ul class="list-inline">
-                            @foreach (var provider in Model.View.VisibleExternalProviders)
+        <div class="col-sm-6">
+            <div class="card">
+                <div class="card-header">
+                    <h2>External Account</h2>
+                </div>
+                <div class="card-body">
+                    <ul class="list-inline">
+                        @foreach (var provider in Model.View.VisibleExternalProviders)
                             {
-                                <li class="list-inline-item">
-                                    <a class="btn btn-secondary"
-                                       asp-page="/ExternalLogin/Challenge"
-                                       asp-route-scheme="@provider.AuthenticationScheme"
-                                       asp-route-returnUrl="@Model.Input.ReturnUrl">
-                                        @provider.DisplayName
-                                    </a>
-                                </li>
+                        <li class="list-inline-item">
+                            <a class="btn btn-secondary"
+                               asp-page="/ExternalLogin/Challenge"
+                               asp-route-scheme="@provider.AuthenticationScheme"
+                               asp-route-returnUrl="@Model.Input.ReturnUrl">
+                                @provider.DisplayName
+                            </a>
+                        </li>
                             }
-                        </ul>
-                    </div>
+                    </ul>
                 </div>
             </div>
+        </div>
         }
 
         @if (!Model.View.EnableLocalLogin && !Model.View.VisibleExternalProviders.Any())
         {
-            <div class="alert alert-warning">
-                <strong>Invalid login request</strong>
-                There are no login schemes configured for this request.
-            </div>
+        <div class="alert alert-warning">
+            <strong>Invalid login request</strong>
+            There are no login schemes configured for this request.
+        </div>
         }
     </div>
 </div>
\ No newline at end of file
diff --git a/Pages/Account/Login/Index.cshtml.cs b/Pages/Account/Login/Index.cshtml.cs
index afc5eab..ff16503 100644
--- a/Pages/Account/Login/Index.cshtml.cs
+++ b/Pages/Account/Login/Index.cshtml.cs
@@ -9,7 +9,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Login;
+namespace BellwoodAuthServer.Pages.Login;
 
 [SecurityHeaders]
 [AllowAnonymous]
diff --git a/Pages/Account/Login/InputModel.cs b/Pages/Account/Login/InputModel.cs
index c869859..823b5b8 100644
--- a/Pages/Account/Login/InputModel.cs
+++ b/Pages/Account/Login/InputModel.cs
@@ -1,6 +1,6 @@
 using System.ComponentModel.DataAnnotations;
 
-namespace QuickstartAuthServer.Pages.Login;
+namespace BellwoodAuthServer.Pages.Login;
 
 public class InputModel
 {
diff --git a/Pages/Account/Login/LoginOptions.cs b/Pages/Account/Login/LoginOptions.cs
index a679040..3cd6b72 100644
--- a/Pages/Account/Login/LoginOptions.cs
+++ b/Pages/Account/Login/LoginOptions.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Login;
+namespace BellwoodAuthServer.Pages.Login;
 
 public static class LoginOptions
 {
diff --git a/Pages/Account/Login/ViewModel.cs b/Pages/Account/Login/ViewModel.cs
index e7be528..ad6c39b 100644
--- a/Pages/Account/Login/ViewModel.cs
+++ b/Pages/Account/Login/ViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Login;
+namespace BellwoodAuthServer.Pages.Login;
 
 public class ViewModel
 {
diff --git a/Pages/Account/Logout/Index.cshtml b/Pages/Account/Logout/Index.cshtml
index dca7bbd..c5d555a 100644
--- a/Pages/Account/Logout/Index.cshtml
+++ b/Pages/Account/Logout/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Logout.Index
+@model BellwoodAuthServer.Pages.Logout.Index
 
 <div class="logout-page">
     <div class="lead">
@@ -8,8 +8,8 @@
     </div>
 
     <form asp-page="/Account/Logout/Index">
-        <input type="hidden" name="logoutId" value="@Model.LogoutId"  />
-        
+        <input type="hidden" name="logoutId" value="@Model.LogoutId" />
+
         <div class="form-group">
             <button class="btn btn-primary">Yes</button>
         </div>
diff --git a/Pages/Account/Logout/Index.cshtml.cs b/Pages/Account/Logout/Index.cshtml.cs
index 6bf7ef2..5d95c42 100644
--- a/Pages/Account/Logout/Index.cshtml.cs
+++ b/Pages/Account/Logout/Index.cshtml.cs
@@ -7,7 +7,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Logout;
+namespace BellwoodAuthServer.Pages.Logout;
 
 [SecurityHeaders]
 [AllowAnonymous]
diff --git a/Pages/Account/Logout/LoggedOut.cshtml b/Pages/Account/Logout/LoggedOut.cshtml
index 993d541..dfd9c55 100644
--- a/Pages/Account/Logout/LoggedOut.cshtml
+++ b/Pages/Account/Logout/LoggedOut.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Logout.LoggedOut
+@model BellwoodAuthServer.Pages.Logout.LoggedOut
 
 <div class="logged-out-page">
     <h1>
@@ -9,15 +9,15 @@
 
     @if (Model.View.PostLogoutRedirectUri != null)
     {
-        <div>
-            Click <a class="PostLogoutRedirectUri" href="@Model.View.PostLogoutRedirectUri">here</a> to return to the
-            <span>@Model.View.ClientName</span> application.
-        </div>
+    <div>
+        Click <a class="PostLogoutRedirectUri" href="@Model.View.PostLogoutRedirectUri">here</a> to return to the
+        <span>@Model.View.ClientName</span> application.
+    </div>
     }
 
     @if (Model.View.SignOutIframeUrl != null)
     {
-        <iframe width="0" height="0" class="signout" src="@Model.View.SignOutIframeUrl"></iframe>
+    <iframe width="0" height="0" class="signout" src="@Model.View.SignOutIframeUrl"></iframe>
     }
 </div>
 
@@ -25,6 +25,6 @@
 {
     @if (Model.View.AutomaticRedirectAfterSignOut)
     {
-        <script src="~/js/signout-redirect.js"></script>
+    <script src="~/js/signout-redirect.js"></script>
     }
 }
\ No newline at end of file
diff --git a/Pages/Account/Logout/LoggedOut.cshtml.cs b/Pages/Account/Logout/LoggedOut.cshtml.cs
index da51261..4ce9c2c 100644
--- a/Pages/Account/Logout/LoggedOut.cshtml.cs
+++ b/Pages/Account/Logout/LoggedOut.cshtml.cs
@@ -2,7 +2,7 @@ using Duende.IdentityServer.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Logout;
+namespace BellwoodAuthServer.Pages.Logout;
 
 [SecurityHeaders]
 [AllowAnonymous]
diff --git a/Pages/Account/Logout/LoggedOutViewModel.cs b/Pages/Account/Logout/LoggedOutViewModel.cs
index 27a7d06..84e079f 100644
--- a/Pages/Account/Logout/LoggedOutViewModel.cs
+++ b/Pages/Account/Logout/LoggedOutViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Logout;
+namespace BellwoodAuthServer.Pages.Logout;
 
 public class LoggedOutViewModel
 {
diff --git a/Pages/Account/Logout/LogoutOptions.cs b/Pages/Account/Logout/LogoutOptions.cs
index 20dbe02..ee988c5 100644
--- a/Pages/Account/Logout/LogoutOptions.cs
+++ b/Pages/Account/Logout/LogoutOptions.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Logout;
+namespace BellwoodAuthServer.Pages.Logout;
 
 public static class LogoutOptions
 {
diff --git a/Pages/Ciba/All.cshtml b/Pages/Ciba/All.cshtml
index f1517e0..8746683 100644
--- a/Pages/Ciba/All.cshtml
+++ b/Pages/Ciba/All.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Ciba.AllModel
+@model BellwoodAuthServer.Pages.Ciba.AllModel
 @{
 }
 
@@ -13,33 +13,33 @@
                 <div class="card-body">
                     @if (Model.Logins.Any())
                     {
-                        <table class="table table-bordered table-striped table-sm">
-                            <thead>
-                                <tr>
-                                    <th>Id</th>
-                                    <th>Client Id</th>
-                                    <th>Binding Message</th>
-                                    <th></th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                @foreach (var login in Model.Logins)
+                    <table class="table table-bordered table-striped table-sm">
+                        <thead>
+                            <tr>
+                                <th>Id</th>
+                                <th>Client Id</th>
+                                <th>Binding Message</th>
+                                <th></th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @foreach (var login in Model.Logins)
                                 {
-                                    <tr>
-                                        <td>@login.InternalId</td>
-                                        <td>@login.Client.ClientId</td>
-                                        <td>@login.BindingMessage</td>
-                                        <td>
-                                            <a asp-page="Consent" asp-route-id="@login.InternalId" class="btn btn-primary">Process</a>
-                                        </td>
-                                    </tr>
+                            <tr>
+                                <td>@login.InternalId</td>
+                                <td>@login.Client.ClientId</td>
+                                <td>@login.BindingMessage</td>
+                                <td>
+                                    <a asp-page="Consent" asp-route-id="@login.InternalId" class="btn btn-primary">Process</a>
+                                </td>
+                            </tr>
                                 }
-                            </tbody>
-                        </table>
+                        </tbody>
+                    </table>
                     }
                     else
                     {
-                        <div>No Pending Login Requests</div>
+                    <div>No Pending Login Requests</div>
                     }
                 </div>
             </div>
diff --git a/Pages/Ciba/All.cshtml.cs b/Pages/Ciba/All.cshtml.cs
index 772a69e..d60877d 100644
--- a/Pages/Ciba/All.cshtml.cs
+++ b/Pages/Ciba/All.cshtml.cs
@@ -3,7 +3,7 @@ using Duende.IdentityServer.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 [SecurityHeaders]
 [Authorize]
diff --git a/Pages/Ciba/Consent.cshtml b/Pages/Ciba/Consent.cshtml
index 15e0c99..67c57a5 100644
--- a/Pages/Ciba/Consent.cshtml
+++ b/Pages/Ciba/Consent.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Ciba.Consent
+@model BellwoodAuthServer.Pages.Ciba.Consent
 @{
 }
 
@@ -7,14 +7,14 @@
     <div class="lead">
         @if (Model.View.ClientLogoUrl != null)
         {
-            <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
+        <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
         }
         <h1>
             @Model.View.ClientName
             <small class="text-muted">is requesting your permission</small>
         </h1>
-        
-        <h3>Verify that this identifier matches what the client is displaying: <em  class="text-primary">@Model.View.BindingMessage</em></h3>
+
+        <h3>Verify that this identifier matches what the client is displaying: <em class="text-primary">@Model.View.BindingMessage</em></h3>
 
         <p>Uncheck the permissions you do not wish to grant.</p>
     </div>
@@ -31,38 +31,38 @@
             <div class="col-sm-8">
                 @if (Model.View.IdentityScopes.Any())
                 {
-                    <div class="form-group">
-                        <div class="card">
-                            <div class="card-header">
-                                <span class="glyphicon glyphicon-user"></span>
-                                Personal Information
-                            </div>
-                            <ul class="list-group list-group-flush">
-                                @foreach (var scope in Model.View.IdentityScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-user"></span>
+                            Personal Information
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.IdentityScopes)
                                 {
-                                    <partial name="_ScopeListItem" model="@scope" />
+                            <partial name="_ScopeListItem" model="@scope" />
                                 }
-                            </ul>
-                        </div>
+                        </ul>
                     </div>
+                </div>
                 }
 
                 @if (Model.View.ApiScopes.Any())
                 {
-                    <div class="form-group">
-                        <div class="card">
-                            <div class="card-header">
-                                <span class="glyphicon glyphicon-tasks"></span>
-                                Application Access
-                            </div>
-                            <ul class="list-group list-group-flush">
-                                @foreach (var scope in Model.View.ApiScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-tasks"></span>
+                            Application Access
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.ApiScopes)
                                 {
-                                    <partial name="_ScopeListItem" model="scope" />
+                            <partial name="_ScopeListItem" model="scope" />
                                 }
-                            </ul>
-                        </div>
+                        </ul>
                     </div>
+                </div>
                 }
 
                 <div class="form-group">
@@ -87,10 +87,10 @@
             <div class="col-sm-4 col-lg-auto">
                 @if (Model.View.ClientUrl != null)
                 {
-                    <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
-                        <span class="glyphicon glyphicon-info-sign"></span>
-                        <strong>@Model.View.ClientName</strong>
-                    </a>
+                <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
+                    <span class="glyphicon glyphicon-info-sign"></span>
+                    <strong>@Model.View.ClientName</strong>
+                </a>
                 }
             </div>
         </div>
diff --git a/Pages/Ciba/Consent.cshtml.cs b/Pages/Ciba/Consent.cshtml.cs
index e4d687c..165bd8c 100644
--- a/Pages/Ciba/Consent.cshtml.cs
+++ b/Pages/Ciba/Consent.cshtml.cs
@@ -7,7 +7,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 [Authorize]
 [SecurityHeaders]
diff --git a/Pages/Ciba/ConsentOptions.cs b/Pages/Ciba/ConsentOptions.cs
index c60aa66..ecc0406 100644
--- a/Pages/Ciba/ConsentOptions.cs
+++ b/Pages/Ciba/ConsentOptions.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 public static class ConsentOptions
 {
diff --git a/Pages/Ciba/Index.cshtml b/Pages/Ciba/Index.cshtml
index 5d95873..76bb02e 100644
--- a/Pages/Ciba/Index.cshtml
+++ b/Pages/Ciba/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Ciba.IndexModel
+@model BellwoodAuthServer.Pages.Ciba.IndexModel
 @{
 }
 
@@ -7,13 +7,13 @@
     <div class="lead">
         @if (Model.LoginRequest.Client.LogoUri != null)
         {
-            <div class="client-logo"><img src="@Model.LoginRequest.Client.LogoUri"></div>
+        <div class="client-logo"><img src="@Model.LoginRequest.Client.LogoUri"></div>
         }
         <h1>
             @Model.LoginRequest.Client.ClientName
             <small class="text-muted">is requesting your permission</small>
         </h1>
-        
+
         <h3>
             Verify that this identifier matches what the client is displaying:
             <em class="text-primary">@Model.LoginRequest.BindingMessage</em>
@@ -25,6 +25,6 @@
         <div>
             <a class="btn btn-primary" asp-page="/Ciba/Consent" asp-route-id="@Model.LoginRequest.InternalId">Yes, Continue</a>
         </div>
-        
+
     </div>
 </div>
diff --git a/Pages/Ciba/Index.cshtml.cs b/Pages/Ciba/Index.cshtml.cs
index 7670d48..a523814 100644
--- a/Pages/Ciba/Index.cshtml.cs
+++ b/Pages/Ciba/Index.cshtml.cs
@@ -4,7 +4,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 [AllowAnonymous]
 [SecurityHeaders]
diff --git a/Pages/Ciba/InputModel.cs b/Pages/Ciba/InputModel.cs
index 35aae79..1df0111 100644
--- a/Pages/Ciba/InputModel.cs
+++ b/Pages/Ciba/InputModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 public class InputModel
 {
diff --git a/Pages/Ciba/ViewModel.cs b/Pages/Ciba/ViewModel.cs
index 396601f..dfad150 100644
--- a/Pages/Ciba/ViewModel.cs
+++ b/Pages/Ciba/ViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Ciba;
+namespace BellwoodAuthServer.Pages.Ciba;
 
 public class ViewModel
 {
diff --git a/Pages/Ciba/_ScopeListItem.cshtml b/Pages/Ciba/_ScopeListItem.cshtml
index 61e4ec1..c6b7698 100644
--- a/Pages/Ciba/_ScopeListItem.cshtml
+++ b/Pages/Ciba/_ScopeListItem.cshtml
@@ -1,4 +1,4 @@
-@using QuickstartAuthServer.Pages.Ciba
+@using BellwoodAuthServer.Pages.Ciba
 @model ScopeViewModel
 
 <li class="list-group-item">
diff --git a/Pages/Consent/ConsentOptions.cs b/Pages/Consent/ConsentOptions.cs
index f232a72..f7aa888 100644
--- a/Pages/Consent/ConsentOptions.cs
+++ b/Pages/Consent/ConsentOptions.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Consent;
+namespace BellwoodAuthServer.Pages.Consent;
 
 public static class ConsentOptions
 {
diff --git a/Pages/Consent/Index.cshtml b/Pages/Consent/Index.cshtml
index c4f532a..fea5b30 100644
--- a/Pages/Consent/Index.cshtml
+++ b/Pages/Consent/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Consent.Index
+@model BellwoodAuthServer.Pages.Consent.Index
 @{
 }
 
@@ -7,7 +7,7 @@
     <div class="lead">
         @if (Model.View.ClientLogoUrl != null)
         {
-            <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
+        <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
         }
         <h1>
             @Model.View.ClientName
@@ -28,38 +28,38 @@
             <div class="col-sm-8">
                 @if (Model.View.IdentityScopes.Any())
                 {
-                    <div class="form-group">
-                        <div class="card">
-                            <div class="card-header">
-                                <span class="glyphicon glyphicon-user"></span>
-                                Personal Information
-                            </div>
-                            <ul class="list-group list-group-flush">
-                                @foreach (var scope in Model.View.IdentityScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-user"></span>
+                            Personal Information
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.IdentityScopes)
                                 {
-                                    <partial name="_ScopeListItem" model="@scope" />
+                            <partial name="_ScopeListItem" model="@scope" />
                                 }
-                            </ul>
-                        </div>
+                        </ul>
                     </div>
+                </div>
                 }
 
                 @if (Model.View.ApiScopes.Any())
                 {
-                    <div class="form-group">
-                        <div class="card">
-                            <div class="card-header">
-                                <span class="glyphicon glyphicon-tasks"></span>
-                                Application Access
-                            </div>
-                            <ul class="list-group list-group-flush">
-                                @foreach (var scope in Model.View.ApiScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-tasks"></span>
+                            Application Access
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.ApiScopes)
                                 {
-                                    <partial name="_ScopeListItem" model="scope" />
+                            <partial name="_ScopeListItem" model="scope" />
                                 }
-                            </ul>
-                        </div>
+                        </ul>
                     </div>
+                </div>
                 }
 
                 <div class="form-group">
@@ -76,14 +76,14 @@
 
                 @if (Model.View.AllowRememberConsent)
                 {
-                    <div class="form-group">
-                        <div class="form-check">
-                            <input class="form-check-input" asp-for="Input.RememberConsent">
-                            <label class="form-check-label" asp-for="Input.RememberConsent">
-                                <strong>Remember My Decision</strong>
-                            </label>
-                        </div>
+                <div class="form-group">
+                    <div class="form-check">
+                        <input class="form-check-input" asp-for="Input.RememberConsent">
+                        <label class="form-check-label" asp-for="Input.RememberConsent">
+                            <strong>Remember My Decision</strong>
+                        </label>
                     </div>
+                </div>
                 }
             </div>
         </div>
@@ -96,10 +96,10 @@
             <div class="col-sm-4 col-lg-auto">
                 @if (Model.View.ClientUrl != null)
                 {
-                    <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
-                        <span class="glyphicon glyphicon-info-sign"></span>
-                        <strong>@Model.View.ClientName</strong>
-                    </a>
+                <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
+                    <span class="glyphicon glyphicon-info-sign"></span>
+                    <strong>@Model.View.ClientName</strong>
+                </a>
                 }
             </div>
         </div>
diff --git a/Pages/Consent/Index.cshtml.cs b/Pages/Consent/Index.cshtml.cs
index 96e1263..503bf66 100644
--- a/Pages/Consent/Index.cshtml.cs
+++ b/Pages/Consent/Index.cshtml.cs
@@ -8,7 +8,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Consent;
+namespace BellwoodAuthServer.Pages.Consent;
 
 [Authorize]
 [SecurityHeaders]
diff --git a/Pages/Consent/InputModel.cs b/Pages/Consent/InputModel.cs
index d0dcc94..156a7ff 100644
--- a/Pages/Consent/InputModel.cs
+++ b/Pages/Consent/InputModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Consent;
+namespace BellwoodAuthServer.Pages.Consent;
 
 public class InputModel
 {
diff --git a/Pages/Consent/ViewModel.cs b/Pages/Consent/ViewModel.cs
index 86c0bcf..61f0cb6 100644
--- a/Pages/Consent/ViewModel.cs
+++ b/Pages/Consent/ViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Consent;
+namespace BellwoodAuthServer.Pages.Consent;
 
 public class ViewModel
 {
diff --git a/Pages/Consent/_ScopeListItem.cshtml b/Pages/Consent/_ScopeListItem.cshtml
index eb6f010..a05fb9f 100644
--- a/Pages/Consent/_ScopeListItem.cshtml
+++ b/Pages/Consent/_ScopeListItem.cshtml
@@ -1,4 +1,4 @@
-@using QuickstartAuthServer.Pages.Consent
+@using BellwoodAuthServer.Pages.Consent
 @model ScopeViewModel
 
 <li class="list-group-item">
diff --git a/Pages/Device/DeviceOptions.cs b/Pages/Device/DeviceOptions.cs
index fe6e290..d879e4b 100644
--- a/Pages/Device/DeviceOptions.cs
+++ b/Pages/Device/DeviceOptions.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Device;
+namespace BellwoodAuthServer.Pages.Device;
 
 public static class DeviceOptions
 {
diff --git a/Pages/Device/Index.cshtml b/Pages/Device/Index.cshtml
index 59f2b5e..fd7c011 100644
--- a/Pages/Device/Index.cshtml
+++ b/Pages/Device/Index.cshtml
@@ -1,141 +1,141 @@
 @page
-@model QuickstartAuthServer.Pages.Device.Index
+@model BellwoodAuthServer.Pages.Device.Index
 @{
 }
 
 @if (Model.Input.UserCode == null)
 {
-    @*We need to collect the user code*@
-    <div class="page-device-code">
-        <div class="lead">
-            <h1>User Code</h1>
-            <p>Please enter the code displayed on your device.</p>
-        </div>
+@*We need to collect the user code*@
+<div class="page-device-code">
+    <div class="lead">
+        <h1>User Code</h1>
+        <p>Please enter the code displayed on your device.</p>
+    </div>
 
-        <div class="row">
-            <div class="col-sm-8">
-                <partial name="_ValidationSummary" />
-            </div>
+    <div class="row">
+        <div class="col-sm-8">
+            <partial name="_ValidationSummary" />
         </div>
+    </div>
 
-        <div class="row">
-            <div class="col-sm-6">
-                <form asp-page="/Device/Index" method="get">
-                    <div class="form-group">
-                        <label for="userCode">User Code:</label>
-                        <input class="form-control" id="userCode" name="userCode" autofocus />
-                    </div>
+    <div class="row">
+        <div class="col-sm-6">
+            <form asp-page="/Device/Index" method="get">
+                <div class="form-group">
+                    <label for="userCode">User Code:</label>
+                    <input class="form-control" id="userCode" name="userCode" autofocus />
+                </div>
 
-                    <button class="btn btn-primary" name="button">Submit</button>
-                </form>
-            </div>
+                <button class="btn btn-primary" name="button">Submit</button>
+            </form>
         </div>
     </div>
+</div>
 }
 else
 {
-    @*collect consent for the user code provided*@
-    <div class="page-device-confirmation">
-        <div class="lead">
-            @if (Model.View.ClientLogoUrl != null)
+@*collect consent for the user code provided*@
+<div class="page-device-confirmation">
+    <div class="lead">
+        @if (Model.View.ClientLogoUrl != null)
             {
-                <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
+        <div class="client-logo"><img src="@Model.View.ClientLogoUrl"></div>
             }
-            <h1>
-                @Model.View.ClientName
-                <small class="text-muted">is requesting your permission</small>
-            </h1>
-            <p>Please confirm that the authorization request matches the code: <strong>@Model.Input.UserCode</strong>.</p>
-            <p>Uncheck the permissions you do not wish to grant.</p>
+        <h1>
+            @Model.View.ClientName
+            <small class="text-muted">is requesting your permission</small>
+        </h1>
+        <p>Please confirm that the authorization request matches the code: <strong>@Model.Input.UserCode</strong>.</p>
+        <p>Uncheck the permissions you do not wish to grant.</p>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-8">
+            <partial name="_ValidationSummary" />
         </div>
+    </div>
 
+    <form asp-page="/Device/Index">
+        <input asp-for="Input.UserCode" type="hidden" />
         <div class="row">
             <div class="col-sm-8">
-                <partial name="_ValidationSummary" />
-            </div>
-        </div>
-
-        <form asp-page="/Device/Index">
-            <input asp-for="Input.UserCode" type="hidden" />
-            <div class="row">
-                <div class="col-sm-8">
-                    @if (Model.View.IdentityScopes.Any())
+                @if (Model.View.IdentityScopes.Any())
                     {
-                        <div class="form-group">
-                            <div class="card">
-                                <div class="card-header">
-                                    <span class="glyphicon glyphicon-user"></span>
-                                    Personal Information
-                                </div>
-                                <ul class="list-group list-group-flush">
-                                    @foreach (var scope in Model.View.IdentityScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-user"></span>
+                            Personal Information
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.IdentityScopes)
                                     {
-                                        <partial name="_ScopeListItem" model="@scope" />
+                            <partial name="_ScopeListItem" model="@scope" />
                                     }
-                                </ul>
-                            </div>
-                        </div>
+                        </ul>
+                    </div>
+                </div>
                     }
 
-                    @if (Model.View.ApiScopes.Any())
+                @if (Model.View.ApiScopes.Any())
                     {
-                        <div class="form-group">
-                            <div class="card">
-                                <div class="card-header">
-                                    <span class="glyphicon glyphicon-tasks"></span>
-                                    Application Access
-                                </div>
-                                <ul class="list-group list-group-flush">
-                                    @foreach (var scope in Model.View.ApiScopes)
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-tasks"></span>
+                            Application Access
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            @foreach (var scope in Model.View.ApiScopes)
                                     {
-                                        <partial name="_ScopeListItem" model="scope" />
+                            <partial name="_ScopeListItem" model="scope" />
                                     }
-                                </ul>
-                            </div>
-                        </div>
+                        </ul>
+                    </div>
+                </div>
                     }
 
-                    <div class="form-group">
-                        <div class="card">
-                            <div class="card-header">
-                                <span class="glyphicon glyphicon-pencil"></span>
-                                Description
-                            </div>
-                            <div class="card-body">
-                                <input class="form-control" placeholder="Description or name of device" asp-for="Input.Description" autofocus>
-                            </div>
+                <div class="form-group">
+                    <div class="card">
+                        <div class="card-header">
+                            <span class="glyphicon glyphicon-pencil"></span>
+                            Description
+                        </div>
+                        <div class="card-body">
+                            <input class="form-control" placeholder="Description or name of device" asp-for="Input.Description" autofocus>
                         </div>
                     </div>
+                </div>
 
-                    @if (Model.View.AllowRememberConsent)
+                @if (Model.View.AllowRememberConsent)
                     {
-                        <div class="form-group">
-                            <div class="form-check">
-                                <input class="form-check-input" asp-for="Input.RememberConsent">
-                                <label class="form-check-label" asp-for="Input.RememberConsent">
-                                    <strong>Remember My Decision</strong>
-                                </label>
-                            </div>
-                        </div>
-                    }
+                <div class="form-group">
+                    <div class="form-check">
+                        <input class="form-check-input" asp-for="Input.RememberConsent">
+                        <label class="form-check-label" asp-for="Input.RememberConsent">
+                            <strong>Remember My Decision</strong>
+                        </label>
+                    </div>
                 </div>
+                    }
             </div>
+        </div>
 
-            <div class="row">
-                <div class="col-sm-4">
-                    <button name="Input.button" value="yes" class="btn btn-primary" autofocus>Yes, Allow</button>
-                    <button name="Input.button" value="no" class="btn btn-secondary">No, Do Not Allow</button>
-                </div>
-                <div class="col-sm-4 col-lg-auto">
-                    @if (Model.View.ClientUrl != null)
+        <div class="row">
+            <div class="col-sm-4">
+                <button name="Input.button" value="yes" class="btn btn-primary" autofocus>Yes, Allow</button>
+                <button name="Input.button" value="no" class="btn btn-secondary">No, Do Not Allow</button>
+            </div>
+            <div class="col-sm-4 col-lg-auto">
+                @if (Model.View.ClientUrl != null)
                     {
-                        <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
-                            <span class="glyphicon glyphicon-info-sign"></span>
-                            <strong>@Model.View.ClientName</strong>
-                        </a>
+                <a class="btn btn-outline-info" href="@Model.View.ClientUrl">
+                    <span class="glyphicon glyphicon-info-sign"></span>
+                    <strong>@Model.View.ClientName</strong>
+                </a>
                     }
-                </div>
             </div>
-        </form>
-    </div>
+        </div>
+    </form>
+</div>
 }
diff --git a/Pages/Device/Index.cshtml.cs b/Pages/Device/Index.cshtml.cs
index 2dd2a8a..fe48329 100644
--- a/Pages/Device/Index.cshtml.cs
+++ b/Pages/Device/Index.cshtml.cs
@@ -10,7 +10,7 @@ using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Microsoft.Extensions.Options;
 
-namespace QuickstartAuthServer.Pages.Device;
+namespace BellwoodAuthServer.Pages.Device;
 
 [SecurityHeaders]
 [Authorize]
diff --git a/Pages/Device/InputModel.cs b/Pages/Device/InputModel.cs
index 887f544..1c1c4a0 100644
--- a/Pages/Device/InputModel.cs
+++ b/Pages/Device/InputModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Device;
+namespace BellwoodAuthServer.Pages.Device;
 
 public class InputModel
 {
diff --git a/Pages/Device/Success.cshtml b/Pages/Device/Success.cshtml
index d2287c0..6e6ffea 100644
--- a/Pages/Device/Success.cshtml
+++ b/Pages/Device/Success.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Device.SuccessModel
+@model BellwoodAuthServer.Pages.Device.SuccessModel
 @{
 }
 
diff --git a/Pages/Device/Success.cshtml.cs b/Pages/Device/Success.cshtml.cs
index 533ebef..36935f2 100644
--- a/Pages/Device/Success.cshtml.cs
+++ b/Pages/Device/Success.cshtml.cs
@@ -1,7 +1,7 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Device;
+namespace BellwoodAuthServer.Pages.Device;
 
 [SecurityHeaders]
 [Authorize]
diff --git a/Pages/Device/ViewModel.cs b/Pages/Device/ViewModel.cs
index 1ffd19f..846af54 100644
--- a/Pages/Device/ViewModel.cs
+++ b/Pages/Device/ViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Device;
+namespace BellwoodAuthServer.Pages.Device;
 
 public class ViewModel
 {
diff --git a/Pages/Device/_ScopeListItem.cshtml b/Pages/Device/_ScopeListItem.cshtml
index 5ada920..c8b7485 100644
--- a/Pages/Device/_ScopeListItem.cshtml
+++ b/Pages/Device/_ScopeListItem.cshtml
@@ -1,4 +1,4 @@
-@using QuickstartAuthServer.Pages.Device
+@using BellwoodAuthServer.Pages.Device
 @model ScopeViewModel
 
 <li class="list-group-item">
diff --git a/Pages/Diagnostics/Index.cshtml b/Pages/Diagnostics/Index.cshtml
index 3757b13..a89c6ba 100644
--- a/Pages/Diagnostics/Index.cshtml
+++ b/Pages/Diagnostics/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Diagnostics.Index
+@model BellwoodAuthServer.Pages.Diagnostics.Index
 
 <div class="diagnostics-page">
     <div class="lead">
@@ -15,18 +15,18 @@
                 <div class="card-body">
                     @if(Model.View.AuthenticateResult.Principal != null)
                     {
-                        <dl>
-                            @foreach (var claim in Model.View.AuthenticateResult.Principal.Claims)
+                    <dl>
+                        @foreach (var claim in Model.View.AuthenticateResult.Principal.Claims)
                             {
-                                <dt>@claim.Type</dt>
-                                <dd>@claim.Value</dd>
+                        <dt>@claim.Type</dt>
+                        <dd>@claim.Value</dd>
                             }
-                        </dl>
+                    </dl>
                     }
                 </div>
             </div>
         </div>
-        
+
         <div class="col">
             <div class="card">
                 <div class="card-header">
@@ -36,28 +36,28 @@
                     <dl>
                         @if (Model.View.AuthenticateResult.Properties != null)
                         {
-                            @foreach (var prop in Model.View.AuthenticateResult.Properties.Items)
+                        @foreach (var prop in Model.View.AuthenticateResult.Properties.Items)
                             {
-                                <dt>@prop.Key</dt>
-                                <dd>@prop.Value</dd>
+                        <dt>@prop.Key</dt>
+                        <dd>@prop.Value</dd>
                             }
                         }
                         @if (Model.View.Clients.Any())
                         {
-                            <dt>Clients</dt>
-                            <dd>
-                                @{
+                        <dt>Clients</dt>
+                        <dd>
+                            @{
                                     var clients = Model.View.Clients.ToArray();
                                     for(var i = 0; i < clients.Length; i++)
                                     {
-                                        <text>@clients[i]</text>
+                                <text>@clients[i]</text>
                                         if (i < clients.Length - 1)
                                         {
-                                            <text>, </text>
+                                <text>, </text>
                                         }
                                     }
-                                }
-                            </dd>
+                            }
+                        </dd>
                         }
                     </dl>
                 </div>
diff --git a/Pages/Diagnostics/Index.cshtml.cs b/Pages/Diagnostics/Index.cshtml.cs
index 80c37b5..2d68b57 100644
--- a/Pages/Diagnostics/Index.cshtml.cs
+++ b/Pages/Diagnostics/Index.cshtml.cs
@@ -3,7 +3,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Diagnostics;
+namespace BellwoodAuthServer.Pages.Diagnostics;
 
 [SecurityHeaders]
 [Authorize]
diff --git a/Pages/Diagnostics/ViewModel.cs b/Pages/Diagnostics/ViewModel.cs
index 4027800..9c70965 100644
--- a/Pages/Diagnostics/ViewModel.cs
+++ b/Pages/Diagnostics/ViewModel.cs
@@ -3,7 +3,7 @@ using System.Text.Json;
 using Duende.IdentityModel;
 using Microsoft.AspNetCore.Authentication;
 
-namespace QuickstartAuthServer.Pages.Diagnostics;
+namespace BellwoodAuthServer.Pages.Diagnostics;
 
 public class ViewModel
 {
diff --git a/Pages/Extensions.cs b/Pages/Extensions.cs
index 7b83616..7bc5741 100644
--- a/Pages/Extensions.cs
+++ b/Pages/Extensions.cs
@@ -3,7 +3,7 @@ using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages;
+namespace BellwoodAuthServer.Pages;
 
 public static class Extensions
 {
diff --git a/Pages/ExternalLogin/Callback.cshtml b/Pages/ExternalLogin/Callback.cshtml
index c9b279c..7634a7e 100644
--- a/Pages/ExternalLogin/Callback.cshtml
+++ b/Pages/ExternalLogin/Callback.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.ExternalLogin.Callback
+@model BellwoodAuthServer.Pages.ExternalLogin.Callback
 
 @{
     Layout = null;
@@ -12,8 +12,8 @@
     <title></title>
 </head>
 <body>
-<div>
-    
-</div>
+    <div>
+
+    </div>
 </body>
 </html>
\ No newline at end of file
diff --git a/Pages/ExternalLogin/Callback.cshtml.cs b/Pages/ExternalLogin/Callback.cshtml.cs
index d42eab9..1dc03b6 100644
--- a/Pages/ExternalLogin/Callback.cshtml.cs
+++ b/Pages/ExternalLogin/Callback.cshtml.cs
@@ -9,7 +9,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.ExternalLogin;
+namespace BellwoodAuthServer.Pages.ExternalLogin;
 
 [AllowAnonymous]
 [SecurityHeaders]
diff --git a/Pages/ExternalLogin/Challenge.cshtml b/Pages/ExternalLogin/Challenge.cshtml
index de6c593..e49d1c4 100644
--- a/Pages/ExternalLogin/Challenge.cshtml
+++ b/Pages/ExternalLogin/Challenge.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.ExternalLogin.Challenge
+@model BellwoodAuthServer.Pages.ExternalLogin.Challenge
 
 @{
     Layout = null;
@@ -12,8 +12,8 @@
     <title></title>
 </head>
 <body>
-<div>
-    
-</div>
+    <div>
+
+    </div>
 </body>
 </html>
\ No newline at end of file
diff --git a/Pages/ExternalLogin/Challenge.cshtml.cs b/Pages/ExternalLogin/Challenge.cshtml.cs
index b059290..f93ed50 100644
--- a/Pages/ExternalLogin/Challenge.cshtml.cs
+++ b/Pages/ExternalLogin/Challenge.cshtml.cs
@@ -4,7 +4,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.ExternalLogin;
+namespace BellwoodAuthServer.Pages.ExternalLogin;
 
 [AllowAnonymous]
 [SecurityHeaders]
diff --git a/Pages/Grants/Index.cshtml b/Pages/Grants/Index.cshtml
index 6949640..2c8454a 100644
--- a/Pages/Grants/Index.cshtml
+++ b/Pages/Grants/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Grants.Index
+@model BellwoodAuthServer.Pages.Grants.Index
 @{
 }
 
@@ -11,80 +11,81 @@
 
     @if (!Model.View.Grants.Any())
     {
-        <div class="row">
-            <div class="col-sm-8">
-                <div class="alert alert-info">
-                    You have not given access to any applications
-                </div>
+    <div class="row">
+        <div class="col-sm-8">
+            <div class="alert alert-info">
+                You have not given access to any applications
             </div>
         </div>
+    </div>
     }
     else
     {
-        foreach (var grant in Model.View.Grants) 
+        foreach (var grant in Model.View.Grants)
         {
-            <div class="card">
-                <div class="card-header">
-                    <div class="row">
-                        <div class="col-sm-8 card-title">
-                            @if (grant.ClientLogoUrl != null)
+    <div class="card">
+        <div class="card-header">
+            <div class="row">
+                <div class="col-sm-8 card-title">
+                    @if (grant.ClientLogoUrl != null)
                             {
-                                <img src="@grant.ClientLogoUrl">
+                    <img src="@grant.ClientLogoUrl">
                             }
-                            <strong>@grant.ClientName</strong>
-                        </div>
+                    <strong>@grant.ClientName</strong>
+                </div>
 
-                        <div class="col-sm-2">
-                            <form asp-page="/Grants/Index">
-                                <input type="hidden" name="clientId" value="@grant.ClientId">
-                                <button class="btn btn-danger">Revoke Access</button>
-                            </form>
-                        </div>
-                    </div>
+                <div class="col-sm-2">
+                    <form asp-page="/Grants/Index">
+                        <input type="hidden" name="clientId" value="@grant.ClientId">
+                        <button class="btn btn-danger">Revoke Access</button>
+                    </form>
                 </div>
-                
-                <ul class="list-group list-group-flush">
-                    @if (grant.Description != null)
+            </div>
+        </div>
+
+        <ul class="list-group list-group-flush">
+            @if (grant.Description != null)
                     {
-                        <li class="list-group-item">
-                            <label>Description:</label> @grant.Description
-                        </li>   
+            <li class="list-group-item">
+                <label>Description:</label> @grant.Description
+            </li>
+
                     }
-                    <li class="list-group-item">
-                        <label>Created:</label> @grant.Created.ToString("yyyy-MM-dd")
-                    </li>
-                    @if (grant.Expires.HasValue)
+            <li class="list-group-item">
+                <label>Created:</label> @grant.Created.ToString("yyyy-MM-dd")
+            </li>
+            @if (grant.Expires.HasValue)
                     {
-                        <li class="list-group-item">
-                            <label>Expires:</label> @grant.Expires.Value.ToString("yyyy-MM-dd")
-                        </li>
+            <li class="list-group-item">
+                <label>Expires:</label> @grant.Expires.Value.ToString("yyyy-MM-dd")
+            </li>
                     }
-                    @if (grant.IdentityGrantNames.Any())
+            @if (grant.IdentityGrantNames.Any())
                     {
-                        <li class="list-group-item">
-                            <label>Identity Grants</label>
-                            <ul>
-                                @foreach (var name in grant.IdentityGrantNames)
+            <li class="list-group-item">
+                <label>Identity Grants</label>
+                <ul>
+                    @foreach (var name in grant.IdentityGrantNames)
                                 {
-                                    <li>@name</li>
+                    <li>@name</li>
                                 }
-                            </ul>
-                        </li>
+                </ul>
+            </li>
                     }
-                    @if (grant.ApiGrantNames.Any())
+            @if (grant.ApiGrantNames.Any())
                     {
-                        <li class="list-group-item">
-                            <label>API Grants</label>
-                            <ul>
-                                @foreach (var name in grant.ApiGrantNames)
+            <li class="list-group-item">
+                <label>API Grants</label>
+                <ul>
+                    @foreach (var name in grant.ApiGrantNames)
                                 {
-                                    <li>@name</li>
+                    <li>@name</li>
                                 }
-                            </ul>
-                        </li>
-                    }
                 </ul>
-            </div>
+            </li>
+                    }
+        </ul>
+    </div>
         }
     }
 </div>
\ No newline at end of file
diff --git a/Pages/Grants/Index.cshtml.cs b/Pages/Grants/Index.cshtml.cs
index ffc4ad4..74cb957 100644
--- a/Pages/Grants/Index.cshtml.cs
+++ b/Pages/Grants/Index.cshtml.cs
@@ -6,7 +6,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Grants;
+namespace BellwoodAuthServer.Pages.Grants;
 
 [SecurityHeaders]
 [Authorize]
diff --git a/Pages/Grants/ViewModel.cs b/Pages/Grants/ViewModel.cs
index 342a6c3..bae68a2 100644
--- a/Pages/Grants/ViewModel.cs
+++ b/Pages/Grants/ViewModel.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages.Grants;
+namespace BellwoodAuthServer.Pages.Grants;
 
 public class ViewModel
 {
diff --git a/Pages/Home/Error/Index.cshtml b/Pages/Home/Error/Index.cshtml
index 3b3e988..9b34ff9 100644
--- a/Pages/Home/Error/Index.cshtml
+++ b/Pages/Home/Error/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Error.Index
+@model BellwoodAuthServer.Pages.Error.Index
 
 <div class="error-page">
     <div class="lead">
@@ -13,22 +13,22 @@
 
                 @if (Model.View.Error != null)
                 {
-                    <strong>
-                        <em>
-                            : @Model.View.Error.Error
-                        </em>
-                    </strong>
+                <strong>
+                    <em>
+                        : @Model.View.Error.Error
+                    </em>
+                </strong>
 
                     if (Model.View.Error.ErrorDescription != null)
                     {
-                        <div>@Model.View.Error.ErrorDescription</div>
+                <div>@Model.View.Error.ErrorDescription</div>
                     }
                 }
             </div>
 
             @if (Model?.View?.Error?.RequestId != null)
             {
-                <div class="request-id">Request Id: @Model.View.Error.RequestId</div>
+            <div class="request-id">Request Id: @Model.View.Error.RequestId</div>
             }
         </div>
     </div>
diff --git a/Pages/Home/Error/Index.cshtml.cs b/Pages/Home/Error/Index.cshtml.cs
index b5bba5b..1649e5a 100644
--- a/Pages/Home/Error/Index.cshtml.cs
+++ b/Pages/Home/Error/Index.cshtml.cs
@@ -2,7 +2,7 @@ using Duende.IdentityServer.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Error;
+namespace BellwoodAuthServer.Pages.Error;
 
 [AllowAnonymous]
 [SecurityHeaders]
diff --git a/Pages/Home/Error/ViewModel.cs b/Pages/Home/Error/ViewModel.cs
index e444e2c..43bd401 100644
--- a/Pages/Home/Error/ViewModel.cs
+++ b/Pages/Home/Error/ViewModel.cs
@@ -1,6 +1,6 @@
 using Duende.IdentityServer.Models;
 
-namespace QuickstartAuthServer.Pages.Error;
+namespace BellwoodAuthServer.Pages.Error;
 
 public class ViewModel
 {
diff --git a/Pages/Index.cshtml b/Pages/Index.cshtml
index be4df8e..7c3d98c 100644
--- a/Pages/Index.cshtml
+++ b/Pages/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Home.Index
+@model BellwoodAuthServer.Pages.Home.Index
 
 <div class="welcome-page">
     <h1>
@@ -35,12 +35,12 @@
 
     @if(Model.License != null)
     {
-        <h2>License</h2>
-        <dl>
-            <dt>Serial Number</dt>
-            <dd>@Model.License.SerialNumber</dd>
-            <dt>Expiration</dt>
-            <dd>@Model.License.Expiration!.Value.ToLongDateString()</dd>
-        </dl>
+    <h2>License</h2>
+    <dl>
+        <dt>Serial Number</dt>
+        <dd>@Model.License.SerialNumber</dd>
+        <dt>Expiration</dt>
+        <dd>@Model.License.Expiration!.Value.ToLongDateString()</dd>
+    </dl>
     }
 </div>
diff --git a/Pages/Index.cshtml.cs b/Pages/Index.cshtml.cs
index 5c585f5..47aeec7 100644
--- a/Pages/Index.cshtml.cs
+++ b/Pages/Index.cshtml.cs
@@ -3,7 +3,7 @@ using Duende.IdentityServer;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Home;
+namespace BellwoodAuthServer.Pages.Home;
 
 [AllowAnonymous]
 public class Index : PageModel
diff --git a/Pages/Log.cs b/Pages/Log.cs
index de080b6..74ada56 100644
--- a/Pages/Log.cs
+++ b/Pages/Log.cs
@@ -1,4 +1,4 @@
-namespace QuickstartAuthServer.Pages;
+namespace BellwoodAuthServer.Pages;
 
 internal static class Log
 {
diff --git a/Pages/Redirect/Index.cshtml b/Pages/Redirect/Index.cshtml
index c4c2e7a..44c6831 100644
--- a/Pages/Redirect/Index.cshtml
+++ b/Pages/Redirect/Index.cshtml
@@ -1,5 +1,5 @@
 @page
-@model QuickstartAuthServer.Pages.Redirect.IndexModel
+@model BellwoodAuthServer.Pages.Redirect.IndexModel
 @{
 }
 
diff --git a/Pages/Redirect/Index.cshtml.cs b/Pages/Redirect/Index.cshtml.cs
index 3c6ab3b..c090f76 100644
--- a/Pages/Redirect/Index.cshtml.cs
+++ b/Pages/Redirect/Index.cshtml.cs
@@ -2,7 +2,7 @@ using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.Redirect;
+namespace BellwoodAuthServer.Pages.Redirect;
 
 [AllowAnonymous]
 public class IndexModel : PageModel
diff --git a/Pages/SecurityHeadersAttribute.cs b/Pages/SecurityHeadersAttribute.cs
index 270a89d..2bb7744 100644
--- a/Pages/SecurityHeadersAttribute.cs
+++ b/Pages/SecurityHeadersAttribute.cs
@@ -1,7 +1,7 @@
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages;
+namespace BellwoodAuthServer.Pages;
 
 public sealed class SecurityHeadersAttribute : ActionFilterAttribute
 {
diff --git a/Pages/ServerSideSessions/Index.cshtml b/Pages/ServerSideSessions/Index.cshtml
index 5c9f4a8..84be9f7 100644
--- a/Pages/ServerSideSessions/Index.cshtml
+++ b/Pages/ServerSideSessions/Index.cshtml
@@ -1,148 +1,148 @@
 @page
-@model QuickstartAuthServer.Pages.ServerSideSessions.IndexModel
+@model BellwoodAuthServer.Pages.ServerSideSessions.IndexModel
 
-    <div class="users-page">
-        <div class="row">
-            <div class="col">
-                <div class="card">
-                    <div class="card-header">
-                        <h2>User Sessions</h2>
-                    </div>
-                
-                    <div class="card-body">
+<div class="users-page">
+    <div class="row">
+        <div class="col">
+            <div class="card">
+                <div class="card-header">
+                    <h2>User Sessions</h2>
+                </div>
+
+                <div class="card-body">
 
                     @if (Model.UserSessions != null)
                     {
-                        <div class="row">
-                            <div class="col-1 text-center">
-                                @if (Model.UserSessions.HasPrevResults)
+                    <div class="row">
+                        <div class="col-1 text-center">
+                            @if (Model.UserSessions.HasPrevResults)
                                 {
-                                    <a class="btn btn-primary" asp-page="/ServerSideSessions/Index"
-                                         asp-route-prev="true" 
-                                         asp-route-token="@Model.UserSessions.ResultsToken"
-                                         asp-route-DisplayNameFilter="@Model.DisplayNameFilter"
-                                         asp-route-SubjectIdFilter="@Model.SubjectIdFilter"
-                                         asp-route-SessionIdFilter="@Model.SessionIdFilter"
-                                    >Prev</a>
+                            <a class="btn btn-primary" asp-page="/ServerSideSessions/Index"
+                               asp-route-prev="true"
+                               asp-route-token="@Model.UserSessions.ResultsToken"
+                               asp-route-DisplayNameFilter="@Model.DisplayNameFilter"
+                               asp-route-SubjectIdFilter="@Model.SubjectIdFilter"
+                               asp-route-SessionIdFilter="@Model.SessionIdFilter">Prev</a>
                                 }
-                            </div>
-                            <div class="col">
-                                <form class="form">
-                                    <div class="form-group row">
-                                        <label asp-for="@Model.DisplayNameFilter" class="col-2 col-form-label">Name:</label>
-                                        <input type="search" asp-for="@Model.DisplayNameFilter" class="col form-control" autofocus />
-                                    </div>
-                                    <div class="form-group row">
-                                        <label asp-for="@Model.SessionIdFilter" class="col-2 col-form-label">Session Id:</label>
-                                        <input type="search" asp-for="@Model.SessionIdFilter" class="col form-control" autofocus />
-                                    </div>
-                                    <div class="form-group row">
-                                        <label asp-for="@Model.SubjectIdFilter" class="col-2 col-form-label">Subject Id:</label>
-                                        <input type="search" asp-for="@Model.SubjectIdFilter" class="col form-control" autofocus />
-                                    </div>
-                                    <div class="form-group row justify-content-end">
-                                        <button type="submit" class="form-control btn btn-success col-1">Filter</button>
-                                    </div>
-                                </form>
-                            </div>
-                            <div class="col-1 text-center">
-                                @if (Model.UserSessions.HasNextResults)
+                        </div>
+                        <div class="col">
+                            <form class="form">
+                                <div class="form-group row">
+                                    <label asp-for="@Model.DisplayNameFilter" class="col-2 col-form-label">Name:</label>
+                                    <input type="search" asp-for="@Model.DisplayNameFilter" class="col form-control" autofocus />
+                                </div>
+                                <div class="form-group row">
+                                    <label asp-for="@Model.SessionIdFilter" class="col-2 col-form-label">Session Id:</label>
+                                    <input type="search" asp-for="@Model.SessionIdFilter" class="col form-control" autofocus />
+                                </div>
+                                <div class="form-group row">
+                                    <label asp-for="@Model.SubjectIdFilter" class="col-2 col-form-label">Subject Id:</label>
+                                    <input type="search" asp-for="@Model.SubjectIdFilter" class="col form-control" autofocus />
+                                </div>
+                                <div class="form-group row justify-content-end">
+                                    <button type="submit" class="form-control btn btn-success col-1">Filter</button>
+                                </div>
+                            </form>
+                        </div>
+                        <div class="col-1 text-center">
+                            @if (Model.UserSessions.HasNextResults)
                                 {
-                                    <a class="btn btn-primary" asp-page="/ServerSideSessions/Index"
-                                         asp-route-token="@Model.UserSessions.ResultsToken"
-                                         asp-route-DisplayNameFilter="@Model.DisplayNameFilter"
-                                         asp-route-SubjectIdFilter="@Model.SubjectIdFilter"
-                                         asp-route-SessionIdFilter="@Model.SessionIdFilter"
-                                    >Next</a>
+                            <a class="btn btn-primary" asp-page="/ServerSideSessions/Index"
+                               asp-route-token="@Model.UserSessions.ResultsToken"
+                               asp-route-DisplayNameFilter="@Model.DisplayNameFilter"
+                               asp-route-SubjectIdFilter="@Model.SubjectIdFilter"
+                               asp-route-SessionIdFilter="@Model.SessionIdFilter">Next</a>
                                 }
-                            </div>
                         </div>
+                    </div>
 
-                        @if (Model.UserSessions.TotalCount.HasValue)
+                    @if (Model.UserSessions.TotalCount.HasValue)
                         {
-                            <div class="text-center">
-                                @if (Model.UserSessions.CurrentPage.HasValue && Model.UserSessions.TotalPages.HasValue)
+                    <div class="text-center">
+                        @if (Model.UserSessions.CurrentPage.HasValue && Model.UserSessions.TotalPages.HasValue)
                                 {
-                                    <text>
-                                        Total Results: @Model.UserSessions.TotalCount,
-                                        Page @Model.UserSessions.CurrentPage of @Model.UserSessions.TotalPages
-                                    </text>
+                        <text>
+                            Total Results: @Model.UserSessions.TotalCount,
+                            Page @Model.UserSessions.CurrentPage of @Model.UserSessions.TotalPages
+                        </text>
                                 }
                                 else
                                 {
-                                    <text>
-                                        Total Results: @Model.UserSessions.TotalCount
-                                    </text>
+                        <text>
+                            Total Results: @Model.UserSessions.TotalCount
+                        </text>
                                 }
-                            </div>
+                    </div>
                         }
 
-                        <br />
-                    
-                        @if (Model.UserSessions.Results.Any())
+                    <br />
+
+                    @if (Model.UserSessions.Results.Any())
                         {
-                            <div>
-                                <table class="table table-bordered table-striped table-sm">
-                                    <thead>
-                                        <tr>
-                                            <th>Subject Id</th>
-                                            <th>Session Id</th>
-                                            <th>Display Name</th>
-                                            <th>Created</th>
-                                            <th>Expires</th>
-                                            <th></th>
-                                        </tr>
-                                    </thead>
-                                    <tbody>
-                                        @foreach (var session in Model.UserSessions.Results)
+                    <div>
+                        <table class="table table-bordered table-striped table-sm">
+                            <thead>
+                                <tr>
+                                    <th>Subject Id</th>
+                                    <th>Session Id</th>
+                                    <th>Display Name</th>
+                                    <th>Created</th>
+                                    <th>Expires</th>
+                                    <th></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                @foreach (var session in Model.UserSessions.Results)
                                         {
-                                            <tr>
-                                                <td>@session.SubjectId</td>
-                                                <td>@session.SessionId</td>
-                                                <td>@session.DisplayName</td>
-                                                <td>@session.Created</td>
-                                                <td>@session.Expires</td>
-                                                <td>
-                                                    <form method="post">
-                                                        <input type="hidden" name="SessionId" value="@session.SessionId" />
-                                                        <button type="submit" class="btn btn-danger">Delete</button>
-                                                    </form>
-                                                </td>
-                                            </tr>
-                                            <tr><td colspan="6">
-                                                <strong>Clients:</strong>
-                                                @if (session.ClientIds?.Any() == true)
+                                <tr>
+                                    <td>@session.SubjectId</td>
+                                    <td>@session.SessionId</td>
+                                    <td>@session.DisplayName</td>
+                                    <td>@session.Created</td>
+                                    <td>@session.Expires</td>
+                                    <td>
+                                        <form method="post">
+                                            <input type="hidden" name="SessionId" value="@session.SessionId" />
+                                            <button type="submit" class="btn btn-danger">Delete</button>
+                                        </form>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td colspan="6">
+                                        <strong>Clients:</strong>
+                                        @if (session.ClientIds?.Any() == true)
                                                 {
-                                                    @(session.ClientIds.Aggregate((x, y) => $"{x}, {y}"))
+                                        @(session.ClientIds.Aggregate((x, y) => $"{x}, {y}"))
                                                 }
                                                 else
                                                 {
-                                                    @("None")
+                                        @("None")
                                                 }
-                                            </td></tr>
+                                    </td>
+                                </tr>
                                         }
-                                    </tbody>
-                                </table>
-                            </div>
+                            </tbody>
+                        </table>
+                    </div>
                         }
                         else
                         {
-                            <div class="text-center">No User Sessions</div>
+                    <div class="text-center">No User Sessions</div>
                         }
                     }
                     else
-                    { 
-                        <div class="row">
-                            <div class="col">
-                                You do not have server-side sessions enabled.
-                                To do so, use <i>AddServerSideSessions</i> on your IdentityServer configuration.
-                                See the <a href="https://docs.duendesoftware.com/identityserver/ui/server-side-sessions">documentation</a> for more information.
-                            </div>
+                    {
+                    <div class="row">
+                        <div class="col">
+                            You do not have server-side sessions enabled.
+                            To do so, use <i>AddServerSideSessions</i> on your IdentityServer configuration.
+                            See the <a href="https://docs.duendesoftware.com/identityserver/ui/server-side-sessions">documentation</a> for more information.
                         </div>
-                    }
                     </div>
+                    }
                 </div>
             </div>
         </div>
     </div>
+</div>
 
diff --git a/Pages/ServerSideSessions/Index.cshtml.cs b/Pages/ServerSideSessions/Index.cshtml.cs
index 1d09850..6407857 100644
--- a/Pages/ServerSideSessions/Index.cshtml.cs
+++ b/Pages/ServerSideSessions/Index.cshtml.cs
@@ -4,7 +4,7 @@ using Duende.IdentityServer.Stores;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace QuickstartAuthServer.Pages.ServerSideSessions;
+namespace BellwoodAuthServer.Pages.ServerSideSessions;
 
 public class IndexModel : PageModel
 {
diff --git a/Pages/Telemetry.cs b/Pages/Telemetry.cs
index deb02c3..37e1aa4 100644
--- a/Pages/Telemetry.cs
+++ b/Pages/Telemetry.cs
@@ -1,6 +1,6 @@
 using System.Diagnostics.Metrics;
 
-namespace QuickstartAuthServer.Pages;
+namespace BellwoodAuthServer.Pages;
 
 #pragma warning disable CA1034 // Nested types should not be visible
 #pragma warning disable CA1724 // Type names should not match namespaces
diff --git a/Pages/TestUsers.cs b/Pages/TestUsers.cs
index 65a5cba..292eb0c 100644
--- a/Pages/TestUsers.cs
+++ b/Pages/TestUsers.cs
@@ -4,7 +4,7 @@ using Duende.IdentityModel;
 using Duende.IdentityServer;
 using Duende.IdentityServer.Test;
 
-namespace QuickstartAuthServer;
+namespace BellwoodAuthServer;
 
 public static class TestUsers
 {
diff --git a/Pages/_ViewImports.cshtml b/Pages/_ViewImports.cshtml
index eb99e93..cdbac67 100644
--- a/Pages/_ViewImports.cshtml
+++ b/Pages/_ViewImports.cshtml
@@ -1,2 +1,2 @@
-???@using QuickstartAuthServer.Pages
+???@using BellwoodAuthServer.Pages
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
